host-bin/mount-chroot

```diff
@@ -60,22 +62,32 @@ addtrap "stty echo 2>/dev/null"
 
 # Function to prompt the user for a passphrase. Sets $passphrase.
 promptNewPassphrase() {
+    NOPASS=''
     echo -n "Choose an encryption passphrase for $NAME: " 1>&2
     [ -t 0 ] && stty -echo
-    while [ -z "$passphrase" ]; do
+    while [ -z "$passphrase" -a -z "$NOPASS" ]; do
         read -r passphrase
-        if [ -z "$passphrase" ]; then
```

Comment:
About this... I wasn't sure if the warning about secure location was enough, or the blank passphrase should be linked to the -k flag. Right now they are independent features. I get the secure location thing, but seeing as the crouton default is no encryption at all, it feels like overkill.